R/derive_params_growth_height.R

```diff
@@ -0,0 +1,282 @@
+#' Derive Anthropometric indicators (Z-Scores/Percentiles-for-Height/Length)
+#' based on Standard Growth Charts
+#'
+#' Derive Anthropometric indicators (Z-Scores/Percentiles-for-Height/Length)
+#' based on Standard Growth Charts for Weight by Height/Length
+#'
+#' @param dataset Input dataset
+#'
+#'   The variables specified in `sex`, `height`, `height_unit`, `parameter`, `analysis_var`
+#'   are expected to be in the dataset.
+#'
+#' @param sex Sex
+#'
+#'   A character vector is expected.
+#'
+#'   Expected Values: `M`, `F`
+#'
+#' @param height Current Height/length
+#'
+#'   A numeric vector is expected. Note that this is the actual height at the current visit.
+#'
+#' @param height_unit Height/Length Unit
+#
+#'   A character vector is expected.
+#'
+#'   Expected values: 'cm'
+#'
+#' @param meta_criteria Metadata dataset
+#'
+#'   A metadata dataset with the following expected variables:
+#'   `HEIGHT_LENGTH`, `HEIGHT_LENGTHU`, `SEX`, `L`, `M`, `S`
+#'
+#'   The dataset can be derived from WHO or user-defined datasets.
+#'   The WHO growth chart metadata datasets are available in the package and will
+#'   require small modifications.
```

Comment:
The original WHO metadata table has height/length increment of 0.5, but the metadata in the admiralpeds packages has increment of 0.1. How was it extrapolated? Do we have any documentation on that?

Reply:
https://www.who.int/tools/child-growth-standards/standards/weight-for-length-height

scroll to the bottom on the `expanded tables` section you will find the increments were by 0.1